packages/odyssey-react/resolveIconSrcPath.js

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+const path = require('path');
+const iconSrc = path.dirname(require.resolve('@okta/odyssey-icons/package.json')) + '/src';
```

Comment:
no need for the package.json to here to resolve t
```suggestion
const iconSrc = path.join(require.resolve('@okta/odyssey-icons'), 'src');
```

Reply:
resolve doesn't work on directories, it needs a module / file.
https://nodejs.org/api/modules.html#modules_require_resolve_request_options

Reply:
ah I see it resolves to the main file specified in the package json good bit of nuance I forgot thank you